README.md

```diff
@@ -1,13 +1,34 @@
 # Project Happy Thoughts API
 
-Replace this readme with your own information about your project.
+Some learing goals for this week included:
+- How to use POST requests to send data to your API
+- How to store data in your database from POST requests
+- How to validate data and ensure your database only contains 'good' data
+- How to build a full API which includes handling of user input
+- How to build an API which works well with an existing frontend
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+These are my endpoints used in frontend: 
+
+app.get: /thoughts (will return array of user post)
+<br>
+app.post: /thoughts (for saving and sending data to API)
+<br>
+app.post: /thoughts/:thoughtId/like (find thought by ID, and update its hearts property to add one like.)
+
+Other endpoints in backend but not used in frontend:
+
+app.get: /thoughts/:thoughtId
+<br>
+app.delete: /thoughts/:id (find thought by ID and delete it)
+<br>
+app.patch: /thoughts/:id (find thought by ID to update property)
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+My main issue this week was starting up a previous project where my API was going to be used. After doing "npm start" outside of the projects' code folder different errors occured. After fixing it and then uploading to Netlify another error came up. I ended up deleting node modules, installing it again and lastly did a git push to update everything both in github and netlify.
```

Comment:
This was a really heavy issue caused by a tiny thing. You shall be really proud of yourself not giving up on this  one.